// Copyright 2023 Ant Group Co., Ltd.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto3";

package secretflow.serving;

// Supported field type.
enum FieldType {
  UNKNOWN_FIELD_TYPE = 0;

  FIELD_BOOL = 1;
  FIELD_INT32 = 2;
  FIELD_INT64 = 3;
  FIELD_FLOAT = 4;
  FIELD_DOUBLE = 5;
  FIELD_STRING = 6;
}
